test/e2e/page-objects/pages/confirmations/redesign/decrypt-message-confirmation.ts

```diff
@@ -0,0 +1,78 @@
+import { Driver } from '../../../../webdriver/driver';
+
+class DecryptMessageConfirmation {
```

Comment:
confirmation classes usually extend to parent classes. I think in this case we could extend this class to Confirmation and then use the this.footerConfirmButton for confirming the decrypt message
Wdyt?

Reply:
see the reply for the above comment, DecryptMessageConfirmation and GetEncryptionKeyConfirmation class don't have the same locator as Confirmation class